catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/DurableFileEndpoint.java

```diff
@@ -192,4 +112,92 @@ public void setFile(File file) {
                     file.toString());
         }
     }
+
+    private static class EventfulFileWrapperGenericFileOperations
+            implements GenericFileOperations<EventfulFileWrapper> {
+        @Override
+        public void setEndpoint(GenericFileEndpoint<EventfulFileWrapper> endpoint) {
+
+        }
+
+        @Override
+        public boolean deleteFile(String name) throws GenericFileOperationFailedException {
+            return new File(name).delete();
+        }
+
+        @Override
+        public boolean existsFile(String name) throws GenericFileOperationFailedException {
+            return new File(name).exists();
+        }
+
+        @Override
+        public boolean renameFile(String from, String to)
+                throws GenericFileOperationFailedException {
+            File srcFile = new File(from);
+            File destFile = new File(to);
+            try {
+                FileUtils.copyFile(srcFile, destFile);
```

Comment:
❓ If this is a rename operation, shouldn't we call `moveFile()` here or delete the source file on completion? Or, should this method be renamed to `copyFile()`?